### PR TITLE
[Readme] fix typo & sync content

### PR DESCRIPTION
**Pull request by KevinZonda**
**Describe the pull request**

- What does your PR fix? Fixes #

- Which triplets are supported/not supported? Have you updated the CI baseline?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
Thanks for the PR!

File: README_zh_CN.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# Vcpkg 总览
+# Vcpkg: 总览
+
+[English Overview](README.md)
 
-## 概要
```
Comment (JackBoosY) [inline README_zh_CN.md:3]: 删掉这里就破坏了目录里概要的链接。

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: README_zh_CN.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
-# Vcpkg 总览
+# Vcpkg: 总览
+
+[English Overview](README.md)
```
Comment (JackBoosY) [inline README_zh_CN.md:3]: 我认为这个不需要，看中文的也不会直接点这个不是？

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: README_zh_CN.md (hunk 2 above)
Comment (KevinZonda) [reply README_zh_CN.md:3]: 我觉得还是有为妙（）
和主文档尽可能保持一致

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**